### DISCUSSION TITLE
Obscure config file

### DISCUSSION THREAD

**Discussion by xFrah** [seymourimadeit/guardvillagers#208] [General]
Could you explain each entry of the config file? It would be useful to understand what they actually do.

**Comment by seymourimadeit** [comment]
What specific config entries do you think need more explaining?

**Reply by xFrah** [reply]
Well some kind of explanation for all of theme would be nice if you have time.

**Reply by seymourimadeit** [reply]
> Well some kind of explanation for all of theme would be nice if you have time.

I can see what I can do about this in the future but the reason I asked for specifics mainly is because I feel like most of them seem self explanatory to me at least. (this is from the guardvillagers-common.toml file)

![png](https://github.com/user-attachments/assets/887a0ddc-bcc1-484a-b090-82df36a1ea1c)
